apps/omg_db/config/config.exs

```diff
@@ -4,10 +4,13 @@ use Mix.Config
 
 config :omg_db,
   # :leveldb, #:ets
-  type: :leveldb,
+  type: :rocksdb,
   # leveldb
   leveldb_path: Path.join([System.get_env("HOME"), ".omg/data"]),
   # leveldb
   server_module: OMG.DB.LevelDB.Server,
   # leveldb
-  server_name: OMG.DB.LevelDB.Server
+  server_name: OMG.DB.LevelDB.Server,
+  r_server_module: OMG.DB.RocksDB.Server,
```

Comment:
I did not want to consolidate the settings now because these will change with the use of poolboy.